.gitignore

```diff
@@ -3,19 +3,34 @@
 /.bundle
 bundle
 /.cache
-/data/version.yml
+/source/_data/versionCache.json
```

Comment:
This file is re-generated on every `yarn build`, and caches (in development) the latest release versions of dart-sass, libsass, and the sass migrator.